<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DrawingLabelTestHarness</class>
 <widget class="QDialog" name="DrawingLabelTestHarness">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>687</width>
    <height>637</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Line Drawing</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item>
    <widget class="QSplitter" name="splitter">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <widget class="QWidget" name="">
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QScrollArea" name="_scrollArea">
         <property name="widgetResizable">
          <bool>true</bool>
         </property>
         <widget class="QWidget" name="scrollAreaWidgetContents">
          <property name="geometry">
           <rect>
            <x>0</x>
            <y>0</y>
            <width>657</width>
            <height>186</height>
           </rect>
          </property>
         </widget>
        </widget>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_4">
         <item>
          <widget class="QCheckBox" name="_labelsBox">
           <property name="text">
            <string>Show Labels</string>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="horizontalSpacer_2">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QDoubleSpinBox" name="_zoomSpinBox">
           <property name="decimals">
            <number>1</number>
           </property>
           <property name="minimum">
            <double>0.100000000000000</double>
           </property>
           <property name="maximum">
            <double>10.000000000000000</double>
           </property>
           <property name="singleStep">
            <double>0.100000000000000</double>
           </property>
           <property name="value">
            <double>1.000000000000000</double>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="label">
           <property name="text">
            <string>Zoom</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_3">
         <item>
          <widget class="QRadioButton" name="_createButton">
           <property name="text">
            <string>Create</string>
           </property>
           <property name="checked">
            <bool>true</bool>
           </property>
           <attribute name="buttonGroup">
            <string notr="true">createCopyGroup</string>
           </attribute>
          </widget>
         </item>
         <item>
          <widget class="QRadioButton" name="_copyButton">
           <property name="text">
            <string>Copy</string>
           </property>
           <property name="checked">
            <bool>false</bool>
           </property>
           <attribute name="buttonGroup">
            <string notr="true">createCopyGroup</string>
           </attribute>
          </widget>
         </item>
         <item>
          <spacer name="horizontalSpacer_3">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_2">
         <item>
          <widget class="QRadioButton" name="_drawButton">
           <property name="text">
            <string>Draw</string>
           </property>
           <property name="checked">
            <bool>true</bool>
           </property>
           <attribute name="buttonGroup">
            <string notr="true">buttonGroup_2</string>
           </attribute>
          </widget>
         </item>
         <item>
          <widget class="QRadioButton" name="_adjustButton">
           <property name="text">
            <string>Adjust</string>
           </property>
           <attribute name="buttonGroup">
            <string notr="true">buttonGroup_2</string>
           </attribute>
          </widget>
         </item>
         <item>
          <spacer name="horizontalSpacer_4">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
     <widget class="QTableWidget" name="_tableWidget">
      <property name="columnCount">
       <number>0</number>
      </property>
     </widget>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QPushButton" name="_saveButton">
       <property name="text">
        <string>Save</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="_calcButton">
       <property name="text">
        <string>Calculate</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="_quitButton">
       <property name="text">
        <string>Quit</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>_quitButton</sender>
   <signal>clicked()</signal>
   <receiver>DrawingLabelTestHarness</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>329</x>
     <y>268</y>
    </hint>
    <hint type="destinationlabel">
     <x>199</x>
     <y>149</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>_drawButton</sender>
   <signal>toggled(bool)</signal>
   <receiver>DrawingLabelTestHarness</receiver>
   <slot>state_toggle()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>74</x>
     <y>303</y>
    </hint>
    <hint type="destinationlabel">
     <x>257</x>
     <y>187</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>_zoomSpinBox</sender>
   <signal>valueChanged(double)</signal>
   <receiver>DrawingLabelTestHarness</receiver>
   <slot>zoom_changed()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>421</x>
     <y>303</y>
    </hint>
    <hint type="destinationlabel">
     <x>257</x>
     <y>187</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>_labelsBox</sender>
   <signal>toggled(bool)</signal>
   <receiver>DrawingLabelTestHarness</receiver>
   <slot>labels_toggled()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>205</x>
     <y>303</y>
    </hint>
    <hint type="destinationlabel">
     <x>257</x>
     <y>187</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>_saveButton</sender>
   <signal>clicked()</signal>
   <receiver>DrawingLabelTestHarness</receiver>
   <slot>save_image()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>69</x>
     <y>343</y>
    </hint>
    <hint type="destinationlabel">
     <x>257</x>
     <y>187</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>_createButton</sender>
   <signal>toggled(bool)</signal>
   <receiver>DrawingLabelTestHarness</receiver>
   <slot>create_copy_toggled()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>299</x>
     <y>258</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>_calcButton</sender>
   <signal>clicked()</signal>
   <receiver>DrawingLabelTestHarness</receiver>
   <slot>calculate()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>190</x>
     <y>553</y>
    </hint>
    <hint type="destinationlabel">
     <x>299</x>
     <y>292</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <buttongroups>
  <buttongroup name="buttonGroup_2"/>
  <buttongroup name="createCopyGroup"/>
 </buttongroups>
</ui>
